mono/tests/bug-60843.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+
+class A : Attribute
+{
+    public object X;
+
+    public static void Main()
+    {
+        var x = (C<int>.E)AttributeTest(typeof(C<>.E));
+        Assert(C<int>.E.V == x);
+        var y = (C<int>.E2[])AttributeTest(typeof(C<>.E2));
+        Assert(y.Length == 2);
+        Assert(y[0] == C<int>.E2.A);
+        Assert(y[1] == C<int>.E2.B);
+    }
+
+    public static object AttributeTest (Type t) {
+        var cas = t.GetCustomAttributes(false);
+        Assert(cas.Length == 1);
+        Assert(cas[0] is A);
+        var a = (A)cas[0];
+        return a.X;
+    }
+
+    private static int AssertCount = 0;
+
+    public static void Assert (
+        bool b, 
+        [CallerFilePath] string sourceFile = null, 
+        [CallerLineNumber] int lineNumber = 0
```

Comment:
FullAOT profiles don't have these attributes:

```
bug-60843.cs(29,10): error CS0246: The type or namespace name 'CallerFilePathAttribute' could not be found (are you missing a using directive or an assembly reference?)
bug-60843.cs(29,10): error CS0246: The type or namespace name 'CallerFilePath' could not be found (are you missing a using directive or an assembly reference?)
bug-60843.cs(30,10): error CS0246: The type or namespace name 'CallerLineNumberAttribute' could not be found (are you missing a using directive or an assembly reference?)
bug-60843.cs(30,10): error CS0246: The type or namespace name 'CallerLineNumber' could not be found (are you missing a using directive or an assembly reference?)
```

Reply:
Weird, thanks for pointing it out

Reply:
Well, that's quite weird. We should add them